Dockerfile

```diff
@@ -1,4 +1,8 @@
 FROM alpine
+
+# less with `--RAW-CONTROL-CHARS` is required for tk show/ tk diff
+RUN RUN apk add --no-cache less
```

Comment:
`RUN RUN`??

Reply:
:man_facepalming: copy pasting seems to be hard. fixed.